appveyor.yml

```diff
@@ -12,29 +12,29 @@ environment:
 
 install:
   - set CMAKE_INSTALL_PREFIX="%APPVEYOR_BUILD_FOLDER%"\build
-  - set PCRE_DEST="%APPVEYOR_BUILD_FOLDER%"\pcre
+  - set PCRE_DEST="%APPVEYOR_BUILD_FOLDER%"\pcre2
   - set CORE_DEST="%APPVEYOR_BUILD_FOLDER%"\editorconfig-core-c
   - set NPP_DEST="%APPVEYOR_BUILD_FOLDER%"
 
   - cmake --version
   - cd "%APPVEYOR_BUILD_FOLDER%"
   # Download PCRE sources
-  - curl -o pcre.zip https://ftp.pcre.org/pub/pcre/pcre-8.41.zip
+  - curl -o pcre.zip https://ftp.pcre.org/pub/pcre/pcre2-10.32.zip
```

Comment:
Why not make this version number adjustable?

Reply:
Done 😁